extensions/websocket/02_websocketstream.js

```diff
@@ -0,0 +1,302 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+"use strict";
+
+((window) => {
+  const core = window.Deno.core;
+  const webidl = window.__bootstrap.webidl;
+
+  webidl.converters["sequence<USVString>"] = webidl.createSequenceConverter(
+    webidl.converters["USVString"],
+  );
+  webidl.converters.AbortSignal = webidl.createInterfaceConverter(
+    "AbortSignal",
+    AbortSignal,
+  );
+  webidl.converters.WebSocketStreamOptions = webidl.createDictionaryConverter(
+    "WebSocketStreamOptions",
+    [
+      {
+        key: "protocols",
+        converter: webidl.converters["sequence<USVString>"],
+        defaultValue: [],
+      },
+      {
+        key: "signal",
+        converter: webidl.converters.AbortSignal,
```

Comment:
Not 100% sure but shouldn't that be:
```suggestion
        converter: webidl.createNullableConverter(
          webidl.converters.AbortSignal),
```

Reply:
No, the property isnt nullable